[2023-01-14T14:22:54.012+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-01-14T14:22:50.915243+00:00 [queued]>
[2023-01-14T14:22:54.024+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-01-14T14:22:50.915243+00:00 [queued]>
[2023-01-14T14:22:54.025+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-14T14:22:54.026+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-01-14T14:22:54.027+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-14T14:22:54.045+0000] {taskinstance.py:1304} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-01-14 14:22:50.915243+00:00
[2023-01-14T14:22:54.052+0000] {standard_task_runner.py:55} INFO - Started process 36083 to run task
[2023-01-14T14:22:54.057+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-01-14T14:22:50.915243+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpaoi2jpym']
[2023-01-14T14:22:54.060+0000] {standard_task_runner.py:83} INFO - Job 37: Subtask is_api_available
[2023-01-14T14:22:54.151+0000] {task_command.py:389} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-01-14T14:22:50.915243+00:00 [running]> on host 35e95982c619
[2023-01-14T14:22:54.240+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-01-14T14:22:50.915243+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-14T14:22:50.915243+00:00
[2023-01-14T14:22:54.242+0000] {http.py:122} INFO - Poking: api/
[2023-01-14T14:22:54.253+0000] {base.py:73} INFO - Using connection ID 'user_api' for task execution.
[2023-01-14T14:22:54.256+0000] {http.py:150} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-01-14T14:22:54.581+0000] {base.py:228} INFO - Success criteria met. Exiting.
[2023-01-14T14:22:54.591+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20230114T142250, start_date=20230114T142254, end_date=20230114T142254
[2023-01-14T14:22:54.631+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-14T14:22:54.679+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
